.github/workflows/scripts/build.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/python3 -u
+
+# -- prioritized --
+import sys
+import os.path
+sys.path.insert(0, os.path.join(os.path.dirname(__file__)))
+
+import common
```

Comment:
maybe rename this folder to a more longer name like `ci_common`? Sometimes it's not so straightforward for readers to distinguish native modules and folders in the same path from names :P